src/dom-updates.js

```diff
@@ -1,44 +1,57 @@
 import {
   getBookings,
-  getTotalCost
+  getTotalCost,
+  searchByDate
 } from './bookings'
 
 import {
   yourBookings,
-  yourBookingsCost
+  yourBookingsCost,
+  dateField,
+  dashboard,
+  searchResults,
+  availableRooms
 } from './scripts'
 
 const setDashboard = (customer, allBookings, allRooms) => {
+  flatpickr(dateField, {
+    minDate: 'today',
+    altInput: true,
+    altFormat: "F j, Y",
+    dateFormat: "Y/m/d"
+  })
+  
   const usersBookings = getBookings(allBookings, customer);
-  const spent = getTotalCost(allRooms, usersBookings);
-
-  yourBookingsCost.innerHTML = '';
-  yourBookings.innerHTML = '';
-
-  yourBookingsCost.innerHTML = `$${spent}`
+  
+  if (typeof usersBookings === 'string') {
+    yourBookingsCost.innerHTML = `<p>${usersBookings}</p>`
+  } else {
+    const spent = getTotalCost(allRooms, usersBookings);
 
-  usersBookings.forEach(booking => {
-    let room = allRooms.find(room => room.number === booking.roomNumber);
-    let image = setRoomImage(room);
-    let bidet = checkForBidet(room);
+    yourBookingsCost.innerHTML = '';
+    yourBookings.innerHTML = '';
 
-    yourBookings.innerHTML += 
-    `
-    <article class="room">
-      <h3 class="room-type">${room.roomType}</h3>  
-      <p class="booking-date">${booking.date}</p>
-      <img class="room-image" src=${image} alt="photo of a ${room.roomType}">
-      <div class="room-details">
-        <span class="num-beds-container">
-          <p class="num-beds">${room.numBeds}</p>
-          <span class="material-icons-round">bed</span>
-        </span>
-        <p class="bed-size">${room.bedSize}</p>
-        <p class="bidet">${bidet}</p>
-      </div>
-    </article>
-    `
-  })
+    yourBookingsCost.innerHTML = `$${spent}`
+    
+    usersBookings.forEach(booking => {
+      let room = allRooms.find(room => room.number === booking.roomNumber);
+      let image = setRoomImage(room);
+      let bidet = checkForBidet(room);
+  
+      yourBookings.innerHTML += 
+      `
+      <article class="room">
+        <h3 class="room-type">${room.roomType} ${bidet}</h3>  
+        <p class="booking-date">${booking.date}</p>
+        <img class="room-image" src=${image} alt="photo of a ${room.roomType}">
+        <div class="room-details">
+          <span class="material-icons-round">bed</span>  
+          <p class="num-beds">${room.numBeds} ${room.bedSize}</p>
+       </div>
+      </article>
+      `
+    })
+  }
 }
 
 const setRoomImage = (room) => {
```

Comment:
Within this function,  you may be able to remove the conditional by replacing it with an object. 
- The keys could be equal to the room type (you may just need quotes around your keys since they would have spaces). 
- The values could be the image names 
- Return something like `./images/${image[room.roomType]}.png`

Reply:
This is so smart! I had yet to come up with a solution that would allow interpolation into the image path because of the spaces in roomType. Yahoo! Thank you very much for this solution. Done and is working perfectly.